py-polars/tests/unit/namespaces/test_datetime.py

```diff
@@ -956,6 +959,9 @@ def test_weekday(time_unit: TimeUnit) -> None:
         "single_dur",
         "spread_even_dur",
         "spread_skewed_dur",
+        "single_time",
+        "spread_even_time",
+        "spread_skewed_time",
```

Comment:
the
```
    if s.dtype == pl.Datetime:
        assert s.median() == expected_median
```
you have on line 973, and that now be extended to dtypes other than `Datetime`? Is it only `pl.Date` that's missing?